gradio/inputs.py

```diff
@@ -42,7 +42,6 @@ class Number(components.Number):
     """
     Component creates a field for user to enter numeric input. Provides a number as an argument to the wrapped function.
     Input type: float
-    Demos: tax_calculator, titanic_survival
```

Comment:
Removing the demos from these deprecated files. We should not be running demos for deprecated components.